### PR TITLE
Add configmap creation

### PR DESCRIPTION
**Pull request by jackzampolin**

### CODE REVIEW

File: examples/gce/README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ This examples runs kube-lego in a separate namespace
 kubectl create -f 05-kube-lego-namespace.yaml
 # Deployment
 kubectl create -f 50-kube-lego-deployment.yaml
+# ConfigMap
+kubectl create -f 20-kube-lego-configmap.yaml
```
Comment (eggsy84) [inline examples/gce/README.md]: I think this has to go on the line before the deployment as the deployment references items in the config map

Comment (jackzampolin) [inline examples/gce/README.md]: :facepalm:

Comment (jackzampolin) [inline examples/gce/README.md]: Fixed

**Comment by simonswine** [comment]
Thanks!

**Pull request merged**